app/models/linkset.rb

```diff
@@ -16,6 +16,11 @@ def empty?
   end
 
   def update_attributes_from_gem_specification!(spec)
-    update!(home: spec.homepage)
+    Links::LINKS.except("changelog", "download").each do |short, long|
+      self.attributes = { short => spec.metadata[long] }
```

Comment:
Is there a way to not hardcode `"changelog", "download"`?

```suggestion
      self.attributes = { short => spec.metadata[long] } if specs.metadata[long]
```

Reply:
Or possibly just assign the values instead of using `attributes?

```suggestion
      public_send("#{short}=", spec.metadata[long] } if respond_to?("#{short}=")
```

Could probably even get away with `respond_to?(short)`